storage/objects/main_test.go

```diff
@@ -224,6 +225,60 @@ func TestKMSObjects(t *testing.T) {
 	}
 }
 
+func TestV4SignedUrl(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var (
+		bucketName = tc.ProjectID + "-signed-url-bucket-name"
+
+		objectName = "foo.txt"
+
+		serviceAccount = os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+	)
+
+	cleanBucket(t, ctx, client, tc.ProjectID, bucketName)
+
+	putUrl, err := generateV4PutObjectSignedUrl(client, bucketName, objectName, serviceAccount)
+	if err != nil {
+		t.Errorf("can't generate a put signed url: %v", err)
+	}
+
+	httpClient := &http.Client{}
+	request, err := http.NewRequest("PUT", putUrl, strings.NewReader("hello world"))
```

Comment:
Maybe create a variable for the input string so it can be used for `ContentLength` and `want` below?

Reply:
It's the way samples are designed right now. I like the idea of showing at least one concrete example instead of making it generic.